articles.tex

```diff
@@ -680,4 +680,36 @@
 The Judicial Board may present an official ruling following the Judicial Investigation.
 This ruling may be appealed in the same manner as an Executive Board decision \ref{Appeals}.
 
+% ARTICLE VII - TEMPORARY AMENDMENTS
+\article{Temporary Amendments}
+In the case that an amendment should be temporary, the amendment's changes should be listed here. 
+Each of these amendments is given a designated period of time, and will cease to exist immediately.
+If at any point interpretation of the changes needs to be made, the Executive Board will use the process defined in \ref{Judicial} to come to an agreement on how to proceed. 
+Occasionaly, Temporary Amendments may become permanent through a referendum, which is described in detail as part of each Temporary Amendment. 
+\asection{Changes to Evaluations}
+During the 2018-2019 period the following will supercede the text otherwise found in this document.
+\asubsection{Changes}
+\asubsubsection{Removal of Introductory Project}
+\ref{The Introductory Project} will no longer be a requirement as described in \ref{The Introductory Process}. 
+Additionally, \ref{Introductory Executive Board} will also no longer be elected or perform any duties during the described period. 
+\asubsubsection{Change of Packet Percentage}
+\ref{Creation of Accounts} currently stipulates that Introductory Members must acquire 60\% (rounded up to the nearest whole person) of required signatures (excluding those of Resident members who have not passed a Membership Evaluation) in the Introductory Packet or successfully complete Introductory Evaluations. 
+During the described period, the percentage of required signatures in the Introductory Packet is increased to 75\%. 
+\asubsubsection{Changes to Packet Times}
+Introductory Members will recieve an \ref{The Introductory Packet} during the second week of each semester. 
+Each Introductory Member finishes packet at the end of week 3. 
```

Comment:
I think that we should specify a hard length of packet. With this wording, it starts somewhere in week 2 (I realize this is being discussed elsewhere also), and ends at the end of week 3. We should state that the packet will end 14 days after it starts.

Reply:
It begins during week 2, and ends at the beginning of week 4 (end of week 3 really.) 

This is intentional wording. It means a packet can be given out at any time during week 2, but always end by the end of week 3.